JS/main.js

```diff
@@ -19,4 +20,128 @@ links.forEach((link) => link.addEventListener('click', () => {
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+
+  },
+  {
+    reverseCard:'reverse-row',
+    cardName:'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy tex.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+    
+  },
+  {
+    reverseCard:'kwacha',
+    cardName:'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+    
+  },
+  {
+    reverseCard:'reverse-row',
+    cardName:'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+    
+  }
+
+]
+
+function popup() {
+  popUp.classList.add('active')
+}
+
+function createCard(cardData){
+  cardcontainer.innerHTML = ` `;
+  cardData.forEach((card, pos) => {
+    const cardMaster = `
+    <li class="card-sec ${card.reverseCard}">
+          <div class="placeholder1">
+            <img
+              class="image1"
+              src="${card.image}"
+              alt="image placeholder"
+            />
+          </div>
+          <div class="card1">
+            <h2 class="post">${card.cardName}</h2>
+            <p class="text">
+              ${card.mainText}
+            </p>
+            <ul class="lang">
+              ${card.cardLang}  
+            </ul>
+            <button class="button pop-${pos}"  type="button">See Project</button>
+          </div>
+        </li>`
+        cardcontainer.insertAdjacentHTML('afterbegin', cardMaster);
+  })
+ 
+} 
+
+
+
+
+createCard(cardsinfo)
+
+const modal = document.querySelector('.pop-up--overlay');
+const openbtn = document.querySelectorAll('.button');
+console.log(openbtn);
+const closebtn = document.querySelector('.close-btn');
+//listen for click
+
+//display popup
+function openModal(){
+  popUp.classList.add('active')
+  if(popUp.classList.contains('active')){
+    document.querySelector('body').style.overflow = 'hidden'
+  }
+  }
+
+
+
+//close popup
+function closeModal(){
+  popUp.classList.remove('active')
+}
```

Comment:
- [ ] **Required:** In trying to fix the page scroll when the modal popup is active, you froze the page permanently. When the modal is closed, kindly unfreeze the main page. You could see [this resource](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollbars#example). For example `document.querySelector("body").style.overflow = "visible";`